src/components/Oauth/OAuthWindow/windowHelpers.tsx

```diff
@@ -0,0 +1,95 @@
+import { useCallback } from 'react';
+
+import { AMP_SERVER, api, Connection } from '../../../services/api';
+
+import { useOAuthWindowToast } from './useOAuthWindowToast';
+
+const DEFAULT_WIDTH = 600; // px
+const DEFAULT_HEIGHT = 600; // px
+
+const SUCCESS_EVENT = 'AUTHORIZATION_SUCCEEDED';
+const FAILURE_EVENT = 'AUTHORIZATION_FAILED';
+
+/**
+ * returns a function to refresh the connection
+ * @param projectId
+ * @param apiKey
+ * @param setSelectedConnection
+ * @returns
+ */
+export function refreshConnection(
```

Comment:
All of the functions in this helper file are higher order functions that return functions, it would be more obvious if they were named like:
- getRefreshConnectionHandler
- getOpenWindowHandler
- getMessageHandler